gemrb/core/Map.cpp

```diff
@@ -2439,7 +2399,15 @@ bool Map::AdjustPositionY(Point &goal, unsigned int radiusx,  unsigned int radiu
 	return false;
 }
 
-void Map::AdjustPosition(Point &goal, unsigned int radiusx, unsigned int radiusy)
+void Map::AdjustPositionNavmap(NavmapPoint &goal, unsigned int radiusx, unsigned int radiusy)
```

Comment:
We're never using the radii, but let's keep it to be consistent with AdjustPosition.

If we do this often, I thought that maybe there's some optimisation potential lurking further down the chain, but I don't see any.